migration/migration1.go

```diff
@@ -0,0 +1,137 @@
+package migration
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/ipfs/go-cid"
+	"github.com/ipfs/go-datastore"
+	"github.com/ipfs/go-datastore/query"
+	"github.com/textileio/powergate/ffs"
+	"github.com/textileio/powergate/util"
+)
+
+// V1MultitenancyMigration contains the logic to upgrade a datastore from
+// version 0 to version 1.
+var V1MultitenancyMigration = Migration(func(txn datastore.Txn) error {
```

Comment:
I think this filename and migrator name is quite clear. 

This is the only migration that exists now. Inside this migration, multiple sub-migrations are run. Recall that this function is an all-or-nothing executing. The "sub-migration" functions is just some name I invented in this comment, from the POV of the `Migrator` it is a single one.